src/PerfView/CommandLineArgs.cs

```diff
@@ -352,7 +352,9 @@ private void SetupCommandLine(CommandLineParser parser)
                 bool hasTpl = false;
                 foreach (var provider in onlyProviders)
                 {
-                    if (0 <= provider.IndexOf(":stack", StringComparison.OrdinalIgnoreCase))
+                    if (0 <= provider.IndexOf("@StacksEnabled=true", StringComparison.OrdinalIgnoreCase))
```

Comment:
:bulb: I believe you can use `Contains` instead of `IndexOf` to make the condition a bit more readable